### PR TITLE
zaki/binary_paris_pages

### PR DESCRIPTION
**Pull request by zakibinary**
Initial commit and updated home page template and styling.

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+firstsource.io
```
Comment (ashkanx) [inline CNAME]: Remove this file

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+baseurl: ""
+url: https://www.firstsource.io/
```
Comment (ashkanx) [inline _config.yml]: remove the link

File: CNAME (hunk 1 above)
Comment (zakibinary) [reply CNAME]: Done

File: _config.yml (hunk 2 above)
Comment (zakibinary) [reply _config.yml]: Done

**Pull request merged**